src/java.base/share/classes/java/lang/Comparable.java

```diff
@@ -83,7 +83,12 @@
  * the class's {@link Object#equals(Object) equals(Object)} method:<pre>
  *     {(x, y) such that x.equals(y)}. </pre><p>
  *
- * This interface is a member of the
+ * In other words, when a class's natural ordering is consistent with
+ * equals, the equivalence classes defined by the equivalence relation
+ * of the {@code equals} method and the equivalence classes defined by
+ * the quotient of the {@code compareTo} method are the same.
+ *
```

Comment:
I think in both cases it should be "the equivalence class defined by...." That is, "equivalence class" should be singular. But there are two of them, so the sentence still properly concludes "... are the same."

Reply:
An equivalence relation defines a *set* of equivalence classes which partition the objects the relation operates on. For example, the "same signum value" equivalence relation on integers has three equivalence classes :
1) negative nonzero numbers (corresponding to signum == -1)
2) zero (corresponding to signum = 0)
3) positive nonzero numbers (corresponding to signum ==1)

Reply:
OK, got it.